services/traction/README.md

```diff
@@ -3,9 +3,16 @@ pip install -r requirements.txt
 gunicorn -k uvicorn.workers.UvicornWorker -b localhost:5000  api.main:app
 ```
 
-
 ```shell
 pip install tox
 tox -e lint
 tox -e test
-```
\ No newline at end of file
+```
+
+how to generate new alembic revision
+
+```docker-compose exec scripts_traciton-api_1 alembic --revision --autogenerate -m "<MIGRATION COMMENT HERE>"
```

Comment:
Spelling?

Reply:
there were multiple issues, this command now actually works. :)